controllers/operators/operators_objects.go

```diff
@@ -0,0 +1,269 @@
+package operators
+
+import (
+	sv1b2 "github.com/RedHatInsights/strimzi-client-go/apis/kafka.strimzi.io/v1beta2"
+
+	gv1alpha1 "github.com/grafana-operator/grafana-operator/v4/api/integreatly/v1alpha1"
+	lv1beta1 "github.com/grafana/loki/operator/api/v1beta1"
+	"github.com/netobserv/network-observability-operator/api/v1alpha1"
+	"github.com/netobserv/network-observability-operator/controllers/constants"
+	"github.com/netobserv/network-observability-operator/controllers/reconcilers"
+	ov1 "github.com/operator-framework/api/pkg/operators/v1"
+	ov1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	pv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func operatorGroup(name string, namespace string) *ov1.OperatorGroup {
+	return &ov1.OperatorGroup{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       constants.OperatorGroup,
+			APIVersion: "operators.coreos.com/v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+			Labels:    map[string]string{"part-of": constants.ObservabilityName},
+		},
+		Spec: ov1.OperatorGroupSpec{
+			TargetNamespaces: []string{namespace},
+		},
+	}
+}
+
+func subscription(name string, namespace string) *ov1alpha1.Subscription {
+	return &ov1alpha1.Subscription{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       constants.SubscriptionKind,
+			APIVersion: "operators.coreos.com/v1alpha1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+			Labels:    map[string]string{"part-of": constants.ObservabilityName},
+		},
+	}
+}
+
+func grafanaSubscription(name string, namespace string) *ov1alpha1.Subscription {
```

Comment:
As discussed in slack, I think it's necessary for productization to make these subscriptions optional, and configurable, in order to adapt upstream vs downstream.
I think a good option would be to externalize these as Subscription yaml resources, that will be easily adaptable between upstream and downstream.

As a consequence, it must be considered in code that subscriptions can potentially be missing (e.g.: no community loki operator), which should not be considered an error.